### PR TITLE
Add commit repo_name and ref columns

### PR DESCRIPTION
**Pull request by Arthelon**
Addresses #25

### CODE REVIEW

File: src/db/models/Commit.ts [hunk 1]
```diff
@@ -8,6 +8,8 @@ export interface Commit {
   score: number;
   message: string;
   committed_at: string;
+  repo_name: string;
+  ref: string;
```
Comment (kiootic) [inline src/db/models/Commit.ts:12]: What is the meaning/usage of this field?

Comment (Arthelon) [reply src/db/models/Commit.ts:12]: For the git ref that the Commit was pushed on

Comment (kiootic) [reply src/db/models/Commit.ts:13]: I agree that repository name is needed, but I can't think of a case where the git ref (branch/tag/etc.) is needed for now. Do you have a scenario in mind?

Comment (Arthelon) [reply src/db/models/Commit.ts:13]: Might be useful if we only want to look at commits from a certain branch (i.e. master) but think this is an unlikely scenario. I can remove this column

Comment (kiootic) [reply src/db/models/Commit.ts:13]: We can remove it for now then.

**Pull request merged**